openpype/tools/sceneinventory/view.py

```diff
@@ -796,3 +796,40 @@ def _show_version_error_dialog(self, version, items):
         ).format(version_str)
         dialog.setText(msg)
         dialog.exec_()
+
+    def update_all(self):
+        """Update all items that are currently 'outdated' in the view"""
+        # Get the source model through the proxy model
+        model = self.model().sourceModel()
```

Comment:
I'm personally not too big of a fan to hardcode this way to get to the source model instead of the proxy. This is solely to preserve the behavior as in the original comment of this PR. E.g. search does not influence the update all, etc.

Reply:
Me too, but both ways are not ideal, this way is at least update logic at one place.

All tools will need to be re-done in a way that their core logic won't be in Qt classes at all so I would not care about perfect solution now.